rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] It would be better if you use getters and setters method for book and person because of the associations. If you need to access a class variable from outside the class, you would need both getters and setters.